frontend/public/i18n.js

```diff
@@ -27,6 +28,9 @@ export const init = () => {
     .init({
       backend: {
         loadPath: '/locales/resource.json?lng={{lng}}&ns={{ns}}',
+        parse: function(data) {
+          return transformNamespace('en', JSON.parse(data));
```

Comment:
Does this work for other languages or only the `en` locale?

Reply:
It looks like `parse` can also take in the language and namespace:

https://github.com/i18next/i18next-http-backend/blob/b1e9823be752f0728b1f84936ee8a2637dc1be26/lib/index.js#L78

We should be able to skip namespaces not prefixed with `plugin__` and pass the correct language.

Reply:
> Does this work for other languages or only the `en` locale?

I don’t know,  and didn’t find any info about it in the README.
However our non-en plurals seem to use `_0`. Not sure if we need to worry about plural conversation for them.

Reply:
Since we have the language available, we should be able to pass simply pass it through. I'm sure there is a reason that the converter needs it, and we don't know what languages plugins might support.

Reply:
Something like this (untested).... Can you give it a try?

```ts
        parse: function(data, lng, ns) {
          const parsed = JSON.parse(data);
          return ns?.startsWith('plugin__') ? transformNamespace(lng, parsed) : parsed;
        }
```

Reply:
> Something like this (untested).... Can you give it a try?
> 
> ```ts
>         parse: function(data, lng, ns) {
>           const parsed = JSON.parse(data);
>           return ns?.startsWith('plugin__') ? transformNamespace(lng, parsed) : parsed;
>         }
> ```

@spadgett Yes, I did something similar but I haven't push to the repo due to a typescript error in https://github.com/openshift/console/blob/master/frontend/public/components/storage-class-form.tsx#L804:

parse: function(data, lng, ns) {
          if (lng && ns.startsWith('plugin__')) {
            return transformNamespace(lng, JSON.parse(data));
          }
        },`
```